helm/sematic-server/values.yaml

```diff
@@ -69,19 +69,19 @@ deployment:
       path: /
       port: http
       scheme: HTTP
-    initialDelaySeconds: 10
+    initialDelaySeconds: 200
     periodSeconds: 5
     successThreshold: 1
     timeoutSeconds: 1
   readiness_probe:
-    failureThreshold: 6
+    failureThreshold: 30
```

Comment:
Seems to be slower to reach readiness now.

Reply:
I added some prints of the time at relevant places, and found this:

- 15 seconds from start of container to end of migration script
- 41 seconds from the time the migration script finishes to the time uvicorn reports that it is listening on port 80
- 2 seconds to do imports in `server.py`
- <1 second to being configuring the ASGI app after imports
- <1 second to configure the ASGI app
- ~1 second from completion of ASGI app configuration to the first successful readiness probe

So the bulk of the time seems to be in the migration script and the uvicorn startup. I'm not sure why uvicorn is so slow to start up, but I don't know that there's anything we could do about it.

As a side note, you might think it would be okay to remove the migration script, since we are already doing it in a helm hook. This is true for folks who are installing via helm, but I'm not sure about people who are just [deploying Sematic as a metadata server](https://docs.sematic.dev/cloud-execution/deploy#deployment-option-1-shared-metadata-server), so it's probably safer to leave it in.